meson_options.txt

```diff
@@ -25,6 +25,8 @@ option('tests', type : 'feature', value : 'enabled', yield : true,
   description: 'Enable or disable unit tests')
 option('introspection', type : 'feature', value : 'disabled', yield : true,
   description : 'Generate gobject-introspection bindings (.gir/.typelib files)')
+option('gtk_doc',  type : 'feature', value : 'auto', yield : true,
```

Comment:
maybe `gtk-doc` instead underscore is more natural as a meson option, I'm not sure and is just a suggestion

Reply:
I don't mind. Personally I prefer to have options with hyphens, but `gtk_doc` is what is used everywhere else. IMHO meson should canonicalise hyphens/underscores anyway, but we're not there yet.

While we're at it, I should also point out that other modules use a boolean option here instead of a feature option. To me a feature option (with explicit auto detection) makes more sense, but again, your call.

Reply:
> but gtk_doc is what is used everywhere else.

Oh I see, even meson documentations prefer that apparently.

> To me a feature option (with explicit auto detection) makes more sense

These all have more to do with my unfamiliarity with meson :)

Reply:
Hopefully addressed in https://github.com/harfbuzz/harfbuzz/pull/2419